CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+-- Added option to input template strings from external file [##1152]
```

Comment:
`##1152` should be just `#1152`

This link needs an expansion at the bottom of the file.